index.js

```diff
@@ -23,7 +23,7 @@ module.exports = {
 		'scss/at-else-if-parentheses-space-before': 'always',
 		'scss/at-function-parentheses-space-before': 'never',
 		'scss/at-function-pattern': [
-			'^([a-z][a-z0-9]*)(-[a-z0-9]+)*$',
+			'^(-?[a-z][a-z0-9]*)(-[a-z0-9]+)*$',
```

Comment:
I was just about to make the same PR 🙂.

All patterns need to include underscore as well. Also, the addition should be _before_ the opening parenthesis:

```suggestion
			'^[-_]?([a-z][a-z0-9]*)(-[a-z0-9]+)*$',
```

https://sass-lang.com/documentation/at-rules/use#private-members

Reply:
Well, this config asks to use kebak-case, which means using `-`, not `_` (`a_b` is also forbidden by this pattern, even though it refers to the same Sass variable than `a-b` due to the underscores being normalized to dashes).

Regarding the parenthesis, I think they are just useless. The rule does not care about capturing groups, and that group is not repeated (unlike the second one)

Reply:
> Well, this config asks to use kebak-case, which means using -, not _

Ah, you're right. I'll reconsider using `_` as prefix for private members then, but to me, it still seems more discoverable than `-`…

Thanks!

Reply:
Well, if you want to do it in your own project, you can reconfigure the pattern.